client/src/main/java/rv/Viewer.java

```diff
@@ -16,11 +16,6 @@
 
 package rv;
 
-import com.jogamp.newt.event.KeyListener;
```

Comment:
The import sections in these files don't like like they were formatted with clang-format 9. You can probably just run that over everything once.

Reply:
resolved in 3bf50f6